corehq/apps/data_dictionary/templates/data_dictionary/partials/confirmation_modals.html

```diff
@@ -117,14 +117,12 @@ <h4 class="modal-title">
             <div class="modal-body">
                 <p>
                     {% blocktrans %}
-                        There are currently no cases that store data with this case property, so it is safe to delete.
+                        There are currently <span id="delete-case-prop-count"></span> that store data with this case property.
                     {% endblocktrans %}
                 </p>
                 <p>
                     {% blocktrans %}
-                    Please note that if a new form or case was updated to the server recently that makes this property
-                    unsafe to delete, this will only be reflected in the Data Dictionary after a short while. This is not
-                    a blocking action and the case property can be created again if needed in the future. Please refer to
+                    This is not a blocking action and the case property can be created again if needed in the future. Please refer to
```

Comment:
You didn't introduce this, but what do you think "blocking action" means? I think you could just start this sentence with "The case property can be created again..."